docs/README.md

```diff
@@ -1,57 +0,0 @@
-# :ledger: Looking for the docs?
```

Comment:
Can we keep the README file?

Reply:
🤦 of course, I got too excited with the `rm` :)